### PR TITLE
helper files for cget.sh and some recipes

### PR DESCRIPTION
**Pull request by jwalt**
this adds some helper files that will be used in subsequent commits

### CODE REVIEW

File: etc/cget/cmake/autotools-build.cmake [hunk 1]
```diff
@@ -0,0 +1,151 @@
+##############################################################################
```
Comment (azoitl) [inline etc/cget/cmake/autotools-build.cmake:1]: please add an eclipse copyright header. You can have a look on CMake files from 4diac FORTE for examples.

**Review by azoitl — CHANGES_REQUESTED** [review]

**Comment by jwalt** [comment]
I have added the corresponding recipes as well to show how the aux files are needed

File: etc/cget/recipes/cross-toolchain/rsync [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
```
Comment (azoitl) [inline etc/cget/recipes/cross-toolchain/rsync:1]: please add a copyright header here as well

**Review by azoitl — CHANGES_REQUESTED** [review]
I'm not sure what to do about the copyright header for the h files. Even if they are generated something should be there.

**Comment by jwalt** [comment]
fixed the license problem and moving recipes to a different pull request

**Review by azoitl — APPROVED** [review]

**Pull request merged**